src/Adapter/Cart/CartPresenter.php

```diff
@@ -212,16 +207,16 @@ public function present(Cart $cart)
 
         $total_excluding_tax = $cart->getOrderTotal(false);
         $total_including_tax = $cart->getOrderTotal(true);
+        $total_discount      = $cart->getOrderTotal(true, Cart::ONLY_DISCOUNTS);
+
+        $subtotals['tax'] = [
+            'type' => 'tax',
+            'label' => $this->translator->trans('Tax', [], 'Cart'),
+            'amount' => $this->pricePresenter->convertAndFormat(
+                $total_including_tax - $total_excluding_tax
+            ),
+        ];
 
-        if ($this->shouldShowTaxLine()) {
```

Comment:
I need this information in OrderPresenter too, and the related template dont take in account this back-office option: what do you suggest ?